sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -61,6 +56,10 @@ def publish_cloud_events(
         :type topic_name: str
         :param body: Array of Cloud Events being published. Required.
         :type body: list[~azure.core.messaging.CloudEvent]
+        :keyword bool binary_mode: Whether to publish a CloudEvent in binary mode. Defaults to False.
+         When specified, the content type is set to `datacontenttype` of the CloudEvent. If 'datacontenttype` is not specified,
```

Comment:
is that supposed to be content_type ?

```suggestion
         When specified, the content type is set to `datacontenttype` of the CloudEvent. If 'content_type` is not specified,
```

Reply:
agree that the wording seems slightly confusing here. Maybe something like the following to clarify: `When True and 'datacontenttype' is specified in CloudEvent, `content_type` is set to 'datacontenttype'
`.